kytos/core/api_server.py

```diff
@@ -198,3 +201,20 @@ def _start_endpoint(self, rule, function, **options):
         self.app.add_url_rule(rule, endpoint, function, **options)
         self.log.info('Started %s - %s', rule,
                       ', '.join(options.get('methods', self.DEFAULT_METHODS)))
+
+    def remove_napp_endpoints(self, napp):
+        """Remove all decorated endpoints.
+
+        Args:
+            napp (Napp): Napp instance to look for rest-decorated methods.
+        """
+        napp_id = f'{napp.username}/{napp.name}'
+        for index, rule in enumerate(self.app.url_map.iter_rules()):
+            if rule.rule.startswith(f'/api/{napp_id}'):
+                self.app.view_functions.pop(rule.endpoint)
+                # pylint: disable=protected-access
+                self.app.url_map._rules.pop(index)
```

Comment:
Are you sure there won't be a problem if we pop while iterating? Maybe you could write a test for this situation.